commune_app/models.py

```diff
@@ -1,3 +1,9 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class Chore(models.Model):
+    # commune=models.ForeignKey(commune,on_delete=models.CASCADE)
```

Comment:
Why is this commented out?

Reply:
waiting for the commune model to be created then i remove the comment